src/core/thread/topology.hpp

```diff
@@ -520,7 +521,13 @@ class Neighbor : public InstanceLocatorInit
  * This class represents a Thread Child.
  *
  */
-class Child : public Neighbor, public IndirectSender::ChildInfo, public DataPollHandler::ChildInfo
+class Child : public Neighbor,
+              public IndirectSender::ChildInfo,
+              public DataPollHandler::ChildInfo
+#if OPENTHREAD_CONFIG_MAC_CSL_TRANSMITTER_ENABLE
+    ,
+              public CslTxScheduler::ChildInfo
```

Comment:
nit: does this formatting work?
```suggestion
class Child : public Neighbor
    , public IndirectSender::ChildInfo
    , public DataPollHandler::ChildInfo
#if OPENTHREAD_CONFIG_MAC_CSL_TRANSMITTER_ENABLE
    , public CslTxScheduler::ChildInfo
```

Reply:
It seems that clang format doesn't like this... `./script/make-pretty clang` changes the format as above.